plugins/modules/ipadnsrecord.py

```diff
@@ -273,7 +321,7 @@
       loc_lat_dir:
         description: LOC Direction Latitude
         required: false
-        choices: ["N", "S"]
+        type: str
```

Comment:
Shouldn't we keep the choices "N" and "S", as only them should be used (RFC 1876)?

Reply:
The `choices` are only in the docmentation section, but are missing in `argument_spec`. If you agree, then I can add them to `argument_spec`.